kminoda_autoware.log

```diff
@@ -0,0 +1,7 @@
+vehicle name is sample
```

Comment:
Could you remove this log file?

Reply:
Sorry it was mistakenly added somehow :bow: 
Fixed: https://github.com/autowarefoundation/autoware_launch/pull/806/commits/f8a56889dc298ef9b00d781e8e21a63b3c08f63e